### PR TITLE
AST-1889 - PHP notice while importing the settings - Undefined index astra-addons

### PR DESCRIPTION
**Pull request by nileshc-bsf**
### Description
- PHP Notice displayed when importing the customizer settings.

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Steps to Reproduce:
- Install fresh Astra theme no Astra Addon needed.
- Export the settings
- Add the logs in wp-config.php

define( 'WP_DEBUG', true ); // To make dev environment.
define( 'WP_DEBUG_LOG', true ); // To make debug.log file in wp-content.

- Now import the settings and check the debug log file.

[25-May-2022 07:26:41 UTC] PHP Notice:  Undefined index: astra-addons in C:\Users\niles\projects\wp60\app\public\wp-content\plugins\astra-import-export\inc\classes\class-astra-import-export-loader.php on line 167
[25-May-2022 07:26:41 UTC] PHP Stack trace:
[25-May-2022 07:26:41 UTC] PHP   1. {main}() C:\Users\niles\projects\wp60\app\public\wp-admin\themes.php:0
[25-May-2022 07:26:41 UTC] PHP   2. require_once() C:\Users\niles\projects\wp60\app\public\wp-admin\themes.php:10
[25-May-2022 07:26:41 UTC] PHP   3. do_action() C:\Users\niles\projects\wp60\app\public\wp-admin\admin.php:175
[25-May-2022 07:26:41 UTC] PHP   4. WP_Hook->do_action() C:\Users\niles\projects\wp60\app\public\wp-includes\plugin.php:476
[25-May-2022 07:26:41 UTC] PHP   5. WP_Hook->apply_filters() C:\Users\niles\projects\wp60\app\public\wp-includes\class-wp-hook.php:331
[25-May-2022 07:26:41 UTC] PHP   6. Astra_Import_Export_Loader->import() C:\Users\niles\projects\wp60\app\public\wp-includes\class-wp-hook.php:307

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ Yes! Astra Pro settings that are available with customizer can be imported/expor
 
 ## Changelog ##
 
+### 1.0.6 ###
+- Fix: PHP notice while importing the settings - Undefined index astra-addons
```
Comment (premanshup) [inline README.md:46]:
```suggestion
- Fix: Compatibility with WordPress 6.0.
```

Comment (imnavanath) [reply README.md:46]: Correct me if I wrong @nileshc-bsf, that was old issue as per as I remember. So PHP notice changelog seems to be more valid here. Right ??

Comment (nileshc-bsf) [reply README.md:46]: @imnavanath - Yes Correct no need to modify the changelog, your thoughts @premanshup

Comment (premanshup) [reply README.md:46]: Okay @imnavanath and @nileshc-bsf! We can go ahead with the original changelog, marking this as resolved.

**Review by premanshup — APPROVED** [review]

**Review by imnavanath — APPROVED** [review]

**Pull request merged**